CHANGELOG.md

```diff
@@ -1,41 +1,81 @@
 ## 0.3.0
 
-- Node >=10.x now required.
-- Added getElementsByClassName method.
-- Added Node to the list of exports
-- Added lowercase of åäö in entityMap.
-- Moved existing sources into `lib` subdirectory.
-- Removed `.npmignore` in favor of `files` entry in package.json.
-- More `package.json` refactoring.
-- Replaced CHANGELOG with more rigorous file.
-- Replaced LICENSE with more rigorous file.
-- Removed component.json (deprecated package manager https://github.com/componentjs/guide)
-- `proof` devDep updated to latest.
-- Fixed CI.
+[Commits](https://github.com/xmldom/xmldom/compare/0.2.1...0.3.0)
+
+- **BREAKING** Node >=10.x is now required.
+- **BREAKING** Removed component.json (deprecated package manager https://github.com/componentjs/guide)
+- **BREAKING** Moved existing sources into `lib` subdirectory.
+- Update devDependency `proof`
+- Removed `.npmignore` in favor of `files` entry in `package.json`.
+
+- [Added `Document.getElementsByClassName`](https://github.com/xmldom/xmldom/issues/24).
+- [Added `Node` to the list of exports](https://github.com/xmldom/xmldom/pull/27)
+- [Added lowercase of åäö in `entityMap`](https://github.com/xmldom/xmldom/pull/23).
+- More `package.json` cleanup/refactoring.
+- Moved CHANGELOG to markdown file.
+- Moved LICENSE to markdown file.
+- Fixed travis CI.
 - README updates.
 
 ## 0.2.1
 
-- More package.json refactoring.
+[Commits](https://github.com/xmldom/xmldom/compare/0.2.0...0.2.1)
+
+- Correct `homepage`, `repository` and `bugs` URLs in `package.json`.
 
 ## 0.2.0
 
-- Now publishing under xmldom npm package again; retiring xmldom-alpha.
+[Commits](https://github.com/xmldom/xmldom/compare/v0.1.27...0.2.0)
+
+**Includes all BREAKING changes introduced in [`xmldom-alpha@v0.1.28`](#0128-only-via-xmldom-alpha-npm-package) and in later commits to master by the original authors.**
+
+- **BREAKING?** [removes the `Object.create` check from the `_extends` method of `dom.js` that added a `__proto__` property](https://github.com/xmldom/xmldom/commit/0be2ae910a8a22c9ec2cac042e04de4c04317d2a#diff-7d1c5d97786fdf9af5446a241d0b6d56L19-L22) ().
+- **BREAKING?** [removed code that added a `__proto__` property](https://github.com/xmldom/xmldom/commit/366159a76a181ce9a0d83f5dc48205686cfaf9cc)
```

Comment:
I don't really like `?` in changelog.

I would sometimes add "POSSIBLY BREAKING" to commit messages for changes when I am unsure about this.